src/backend/codegen_cpp.cpp

```diff
@@ -1710,60 +1722,72 @@ namespace graphit {
 	
         auto edgeset_apply_func_name = edgeset_apply_func_gen_->genFunctionName(apply);
         oss << edgeset_apply_func_name << "(";
-        auto mir_var = std::dynamic_pointer_cast<mir::VarExpr>(apply->target);
-        std::vector<std::string> arguments = std::vector<std::string>();
 
+        apply->target->accept(this);
+        oss << ", ";
 
-        if (apply->from_func != "") {
-            if (mir_context_->isFunction(apply->from_func)) {
-                // the schedule is an input from function
-                // Create functor instance
-                arguments.push_back(genFunctorNameAsArgumentString(apply->from_func, apply->fromFuncFunctorArgs));
+        if (apply->from_func) {
```

Comment:
again, this kind of check might not be safe because from_func is not guaranteed to be a nullptr by default. We need to double-check the initialization part?